src/gl/gl_builders.js

```diff
@@ -270,42 +279,200 @@ GLBuilders.buildPolylines = function (
             }
 
             if (isPrev || isNext) {
-                addVertexPair(coordCurr, normCurr, i/lineSize, constants);
+                // If is the BEGINING of a LINE
+                if (i === 0 && !isPrev && !closed_polygon){
```

Comment:
Also, space between `)` and `{` please, e.g. `if (i === 0 && !isPrev && !closed_polygon) {`